pkg/cmd/logs.go

```diff
@@ -0,0 +1,112 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/wercker/stern/kubernetes"
+
+	"github.com/cloud-native-application/rudrx/api/types"
+	cmdutil "github.com/cloud-native-application/rudrx/pkg/cmd/util"
+	"github.com/spf13/cobra"
+	sternCmd "github.com/wercker/stern/cmd"
+	"github.com/wercker/stern/stern"
+)
+
+func NewLogsCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = "logs <appname>"
+	cmd.Short = "Tail pods logs of an application"
+	cmd.Long = "Tail pods logs of an application"
+	cmd.PreRunE = func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			ioStreams.Errorf("please specify the application name")
+		}
+		appName := args[0]
+		cmd.SetArgs([]string{appName + ".*"})
+		env, err := GetEnv(cmd)
+		if err != nil {
+			return err
+		}
+		namespace := env.Namespace
+		cmd.Flags().String("namespace", namespace, "")
+		return nil
+	}
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		config, err := sternCmd.ParseConfig(args)
+		if err != nil {
+			return err
+		}
+		ctx := context.Background()
+		if err := Run(ctx, config, ioStreams); err != nil {
+			return err
+		}
+		return nil
+	}
+	cmd.Annotations = map[string]string{
+		types.TagCommandType: types.TypeApp,
+	}
+	return cmd
+}
+
+// Run refer to the implementation at https://github.com/oam-dev/stern/blob/master/stern/main.go
```

Comment:
This is the reference link for this implementation.

Reply:
refer to the original link : github.com/wercker/stern

Reply:
Done in the latest commit. Thanks.